application-tour-ui/src/main/resources/TourCode/TourJson.xml

```diff
@@ -132,6 +147,5 @@
   #set ($discard = $response.setContentType('application/json'))
   $jsontool.serialize({'tours': $tours})
 #end
-{{/velocity}}
-</content>
+{{/velocity}}</content>
```

Comment:
Please do not commit such changes. It's cosmetic and it hides the interesting changes.